WeakAuras/BossMods.lua

```diff
@@ -1523,11 +1523,16 @@ Private.event_prototypes["Boss Mod Announce"] = {
       type = "string",
       preamble = "local counter = Private.ExecEnv.CreateTriggerCounter(%q)",
       test = "counter:SetCount(tonumber(count) or 0) == nil and counter:Match()",
+      conditionPreamble = function(input)
+        return Private.ExecEnv.CreateTriggerCounter(input)
+      end,
       conditionTest = function(state, needle, op, preamble)
-        return preamble and preamble:Check(state.count)
+        preamble:SetCount(tonumber(state.count) or 0)
+        return preamble:Match()
       end,
       store = true,
-      conditionType = "number",
+      conditionType = "string",
```

Comment:
It should be the same type used for triggers like Combat Log's count, aka "number"

![image](https://github.com/user-attachments/assets/1ebeb0b2-a192-47c5-a0d2-5c8b0e45b4cb)

Reply:
hmm i made it work with the same syntax as the trigger, so you can use e.g. "2,4"

Reply:
it's odd to have them work differently, but i'll let it be